### PR TITLE
fix(lb): enriched S3 failover content MTA-1774

### PR DESCRIPTION
**Pull request by RoRoJ**
Enriched S3 failover doc content

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

**Review by rubenobadia — COMMENTED** [review]
it's good !

File: network/load-balancer/how-to/set-up-s3-failover.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  description: This page explains how to set up an S3 failover for your Load Balancer, using the Scaleway Object Storage Bucket Website feature
+content:
+  h1: How to set up an S3 failover
+  paragraph: This page explains how to set up an S3 failover for your Load Balancer, using the Scaleway Object Storage Bucket Website feature
+dates:
+  validation: 2022-02-21
+  posted: 2022-02-21
+---
+
+The **S3 failover** feature allows you to redirect users to a static website hosted on Scaleway Object storage, in the case that none of your Load Balancer's backends are available. This website could be a simple, single webpage or else something much more complex: you build it according to your own requirements. You can set up an S3 failover during creation of your Load Balancer, or at any point after creation.
```
Comment (ldecarvalho-doc) [inline network/load-balancer/how-to/set-up-s3-failover.mdx:13]:
```suggestion
The **S3 failover** feature allows you to redirect users to a static website hosted on Scaleway Object Storage, in the case that none of your Load Balancer's backends are available. This website could be a simple, single webpage or else something much more complex: you build it according to your own requirements. You can set up an S3 failover during creation of your Load Balancer, or at any point after creation. 
```

File: network/load-balancer/how-to/set-up-s3-failover.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+dates:
+  validation: 2022-02-21
+  posted: 2022-02-21
+---
+
+The **S3 failover** feature allows you to redirect users to a static website hosted on Scaleway Object storage, in the case that none of your Load Balancer's backends are available. This website could be a simple, single webpage or else something much more complex: you build it according to your own requirements. You can set up an S3 failover during creation of your Load Balancer, or at any point after creation. 
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
```
Comment (ldecarvalho-doc) [inline network/load-balancer/how-to/set-up-s3-failover.mdx:17]: you have a bucket with the bucket website feature configured maybe?

File: network/load-balancer/how-to/set-up-s3-failover.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+1. Click **Load Balancers** in the Network section of the [Scaleway Console](https://console.scaleway.com/) side menu. The list of your Load Balancers displays.
+
+2. Click the Load Balancer you wish to set up an S3 failover for. The Load Balancer's **Overview** page displays.
+
+3. Click the **Backends** tab. The list of your backends displays.
+
+4. Click the <Icon name="edit" /> icon next to the backend you wish to set up an S3 failover for. A pop-up displays.
+
+5. Click the  **+Advanced Settings** button.
```
Comment (ldecarvalho-doc) [inline network/load-balancer/how-to/set-up-s3-failover.mdx:49]:
```suggestion
5. Click the  **+ Advanced Settings** button.
```
is the `+` supposed to be here?

**Review by ldecarvalho-doc — APPROVED** [review]

File: network/load-balancer/how-to/set-up-s3-failover.mdx (hunk 3 above)
Comment (RoRoJ) [reply network/load-balancer/how-to/set-up-s3-failover.mdx:49]: yes :-)

**Pull request merged**